test/clj/rems/test_browser.clj

```diff
@@ -883,11 +883,27 @@
              (slurp-fields :form)))
       (testing "preview"
         ;; the text is split into multiple DOM nodes so we need btu/has-text?, :fn/has-text is simpler for some reason
-        (is (btu/has-text? {:tag :label :class :application-field-label :fn/has-text "Text area (EN)"}
-                           "(max 127 characters)"))
-        (is (btu/has-text? {:tag :label :class :application-field-label :fn/has-text "Text area (EN)"}
-                           "(optional)"))
-        (is (btu/visible? {:tag :label :class :application-field-label :fn/has-text "Option list (EN)"}))))
+        (btu/wait-visible {:tag :button :fn/has-class :info-button})
+        (let [button (first (btu/query-all {:tag :button :fn/has-class :info-button}))]
+          (is (btu/has-text? {:tag :label :class :application-field-label :fn/has-text "Text area (EN)"}
+                             "(max 127 characters)"))
+          (is (btu/has-text? {:tag :label :class :application-field-label :fn/has-text "Text area (EN)"}
+                             "(optional)"))
+          (is (btu/visible? {:tag :label :class :application-field-label :fn/has-text "Option list (EN)"}))
```

Comment:
If these were in the original place, there would be no need to read the diff here. These statements don't use the button at all?